rllib/algorithms/crr/crr.py

```diff
@@ -43,7 +42,7 @@ def __init__(self, algo_class=None):
         # __sphinx_doc_end__
         # fmt: on
         self.replay_buffer_config = {
-            "type": "ReplayBuffer",
+            "type": MultiAgentReplayBuffer,
```

Comment:
is this really related??

Reply:
Yeah, we should probably revert this one-line change. Doesn't seem to be related.

I do agree we should go as generic as possible for our algo default configs, however, not sure about the pure offline RL algos (like CRR). I'm assuming multi-agent RL doesn't really work too well for offline RL anyways due to the extreme env dynamics that would need proper exploration (not available offline) to work.